src/runner/runner.go

```diff
@@ -155,14 +158,26 @@ func dumpMetrics(clientID string, debug bool) {
 	}
 
 	if bytesGenerated > 0 {
-		log.Println("Атака проводиться успішно! Руський воєнний корабль іди нахуй!")
-		log.Println("Attack is successful! Russian warship, go fuck yourself!")
-		log.Printf("The app has generated approximately %v bytes of traffic\n", bytesGenerated)
+		fmt.Println()
 
-		if bytesProcessed > 0 {
-			log.Printf("Of which for %v bytes we received some response from the target", bytesProcessed)
-		}
+		fmt.Println("Атака проводиться успішно! Руський воєнний корабль іди нахуй!")
+		fmt.Println("Attack is successful! Russian warship, go fuck yourself!")
+
+		const BytesInMegabytes = 1024
```

Comment:
actually 1024*1024 bytes in one megabyte... 1024 is one kilobyte.

Reply:
Fixed. 
Guess not the best idea to commit at night.